docs/concepts/programming_constructs.md

```diff
@@ -73,6 +73,23 @@ emit -limit 10
       points_3=pow(points, 3)
 ```
 
+Reducers
+========
+
+Reducers are a Juttle-specific construct that allows operating on values of a chosen field, as points stream through the flowgraph, to carry out a running computation.
```

Comment:
Reducers can operate on any field in a point (via `#fieldName`), not just the field name passed in as an argument.